package.json

```diff
@@ -0,0 +1,94 @@
+    "parcel-bundler": "^1.12.4",
+    "sourcegraph": "^24.7.0",
+    "typescript": "^3.9.7"
+  },
+    "icon": "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 70 70'%3E%3ClinearGradient id='a' gradientUnits='userSpaceOnUse' x1='.79' y1='40.089' x2='33.317' y2='40.089'%3E%3Cstop offset='.258' stop-color='%23f97a12'/%3E%3Cstop offset='.459' stop-color='%23b07b58'/%3E%3Cstop offset='.724' stop-color='%23577bae'/%3E%3Cstop offset='.91' stop-color='%231e7ce5'/%3E%3Cstop offset='1' stop-color='%23087cfa'/%3E%3C/linearGradient%3E%3Cpath fill='url(%23a)' d='M17.7 54.6L.8 41.2l8.4-15.6L33.3 35z'/%3E%3ClinearGradient id='b' gradientUnits='userSpaceOnUse' x1='25.767' y1='24.88' x2='79.424' y2='54.57'%3E%3Cstop offset='0' stop-color='%23f97a12'/%3E%3Cstop offset='.072' stop-color='%23cb7a3e'/%3E%3Cstop offset='.154' stop-color='%239e7b6a'/%3E%3Cstop offset='.242' stop-color='%23757b91'/%3E%3Cstop offset='.334' stop-color='%23537bb1'/%3E%3Cstop offset='.432' stop-color='%23387ccc'/%3E%3Cstop offset='.538' stop-color='%23237ce0'/%3E%3Cstop offset='.655' stop-color='%23147cef'/%3E%3Cstop offset='.792' stop-color='%230b7cf7'/%3E%3Cstop offset='1' stop-color='%23087cfa'/%3E%3C/linearGradient%3E%3Cpath fill='url(%23b)' d='M70 18.7l-1.3 40.5L41.8 70 25.6 59.6 49.3 35 38.9 12.3l9.3-11.2z'/%3E%3ClinearGradient id='c' gradientUnits='userSpaceOnUse' x1='63.228' y1='42.915' x2='48.29' y2='-1.719'%3E%3Cstop offset='0' stop-color='%23fe315d'/%3E%3Cstop offset='.078' stop-color='%23cb417e'/%3E%3Cstop offset='.16' stop-color='%239e4e9b'/%3E%3Cstop offset='.247' stop-color='%23755bb4'/%3E%3Cstop offset='.339' stop-color='%235365ca'/%3E%3Cstop offset='.436' stop-color='%23386ddb'/%3E%3Cstop offset='.541' stop-color='%232374e9'/%3E%3Cstop offset='.658' stop-color='%231478f3'/%3E%3Cstop offset='.794' stop-color='%230b7bf8'/%3E%3Cstop offset='1' stop-color='%23087cfa'/%3E%3C/linearGradient%3E%3Cpath fill='url(%23c)' d='M70 18.7L48.7 43.9l-9.8-31.6 9.3-11.2z'/%3E%3ClinearGradient id='d' gradientUnits='userSpaceOnUse' x1='10.72' y1='16.473' x2='55.524' y2='90.58'%3E%3Cstop offset='0' stop-color='%23fe315d'/%3E%3Cstop offset='.04' stop-color='%23f63462'/%3E%3Cstop offset='.104' stop-color='%23df3a71'/%3E%3Cstop offset='.167' stop-color='%23c24383'/%3E%3Cstop offset='.291' stop-color='%23ad4a91'/%3E%3Cstop offset='.55' stop-color='%23755bb4'/%3E%3Cstop offset='.917' stop-color='%231d76ed'/%3E%3Cstop offset='1' stop-color='%23087cfa'/%3E%3C/linearGradient%3E%3Cpath fill='url(%23d)' d='M33.7 58.1L5.6 68.3l4.5-15.8L16 33.1 0 27.7 10.1 0l22 2.7 21.6 24.7z'/%3E%3Cg%3E%3Cpath d='M13.7 13.5h43.2v43.2H13.7z'/%3E%3Cpath fill='%23fff' d='M17.7 48.6h16.2v2.7H17.7zM29.4 22.4v-3.3h-9v3.3H23v11.3h-2.6V37h9v-3.3h-2.5V22.4zM38 37.3c-1.4 0-2.6-.3-3.5-.8-.9-.5-1.7-1.2-2.3-1.9l2.5-2.8c.5.6 1 1 1.5 1.3.5.3 1.1.5 1.7.5.7 0 1.3-.2 1.8-.7.4-.5.6-1.2.6-2.3V19.1h4v11.7c0 1.1-.1 2-.4 2.8-.3.8-.7 1.4-1.3 2-.5.5-1.2 1-2 1.2-.8.3-1.6.5-2.6.5'/%3E%3C/g%3E%3C/svg%3E"
```

Comment:
**question**: Did you run this through SVGOMG?

Reply:
Yes :)